### PR TITLE
mediainfo: add page

### PR DESCRIPTION
**Pull request by agnivade**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/mediainfo.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# mediainfo
+
+> Display metadata from video and audio files.
+
+- Display metadata for a given file in the console:
+
+`mediainfo {{file}}`
+
+- Store the output to a given file apart from displaying in the console:
```
Comment (sbrl) [inline pages/common/mediainfo.md]: "apart from displaying in the console" feels a bit confusing. Does it display the data into the console too? Or not?

Also, the `--Logfile` bit has a capital letter. Was this intentional, or does it not work without it?

File: pages/common/mediainfo.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# mediainfo
+
+> Display metadata from video and audio files.
+
+- Display metadata for a given file in the console:
+
+`mediainfo {{file}}`
```
Comment (sbrl) [inline pages/common/mediainfo.md]: Perhaps `path/to/file.ext` might be more consistent with the style we've used on other pages?

File: pages/common/mediainfo.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+- Store the output to a given file apart from displaying in the console:
+
+`mediainfo --Logfile={{out.txt}} {{file}}`
+
+- Display the list of metadata attributes that can be extracted:
```
Comment (sbrl) [inline pages/common/mediainfo.md]: This is useful - it seems to give a list of parameters and then a description for each. Different file types may have different metadata attributes that you can extract I assume though. Is there a way to list them for a specific file type too?

File: pages/common/mediainfo.md (hunk 1 above)
Comment (agnivade) [reply pages/common/mediainfo.md]: Ah yes, I should change it to "along with displaying in the console".

> Also, the --Logfile bit has a capital letter.

Hmm, it does work with `--logfile`. But `--Logfile` is what it says in its help section. So I would rather be safe.

File: pages/common/mediainfo.md (hunk 3 above)
Comment (agnivade) [reply pages/common/mediainfo.md]: Ah no. I didn't find anything like that.

Comment (sbrl) [reply pages/common/mediainfo.md]: Shame! Just a thought, anyway.

File: pages/common/mediainfo.md (hunk 1 above)
Comment (sbrl) [reply pages/common/mediainfo.md]: Ah, ok. Let's stick with the safe option then.

File: pages/common/mediainfo.md (hunk 2 above)
Comment (agnivade) [reply pages/common/mediainfo.md]: We usually use the `file.ext` syntax when adding the extension makes the command more clear. https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#token-syntax see pt. 5

**Comment by agnivade** [comment]
ping @sbrl

**Comment by sbrl** [comment]
Looks ok to me!

**Pull request merged**